iBudget/group/models.py

```diff
@@ -60,6 +60,50 @@ def get_group_by_id(group_id):
         except Group.DoesNotExist:
             return None
 
+    @staticmethod
+    def filter_groups_by_user_id(user_id):
+        """
+        Args:
+            user_id(int): Current session user`s id.
+        Returns:
+            List of Groups objects .
+
+        """
+        users_groups = []
+        users_groups.extend(Group.objects.filter(members=user_id))
```

Comment:
This is a bad approach because `Group` model in the future can get properties that should not be added to the `users_groups` list. It will work for now but this is a bad smell.

Reply:
The same comment is valid for all similar code.

Reply:
> The same comment is valid for all similar code.

Should I make list with only required model attributes ?

Reply:
Ideally, you need explicitly select only needed values.